src/chrome/content/rules/TT.se.xml

```diff
@@ -1,7 +1,15 @@
 <ruleset name="TT.se">
-<target host="tt.se" />
-<target host="www.tt.se" />
-<rule from="^http://tt\.se/" to="https://www.tt.se/"/>
-<rule from="^http://www\.tt\.se/" to="https://www.tt.se/"/>
-</ruleset>
+	<target host="tt.se" />
+	<target host="www.tt.se" />
+
+	<target host="adm.tt.se" />
+	<target host="api.tt.se" />
+	<target host="app.tt.se" />
+	<target host="beta.tt.se" />
+	<target host="spy.tt.se" />
+	<!-- <target host="text.tt.se" /> -->
```

Comment:
Please add comments to the top and add the reason (incomplete certificate chain)